pom.xml

```diff
@@ -420,6 +420,13 @@
                       <bannedImport>org.apache.commons.logging.LogFactory</bannedImport>
                     </bannedImports>
                   </restrictImports>
+                  <restrictImports implementation="de.skuzzle.enforcer.restrictimports.rule.RestrictImports">
+                    <includeTestCode>true</includeTestCode>
+                    <reason>Do not use phoenix-thirdparty provided Guava</reason>
```

Comment:
Nit: maybe emphasise that's it's for 4.x only.

Reply:
Yeah, makes sense. Thanks